content/tbb/tbb-43/contents.lr

```diff
@@ -0,0 +1,10 @@
+_model: question
+---
+title: What is the difference between using Tor Browser and 'Incognito mode' or private tabs?
+---
+seo_slug: difference-between-tor-browser-incognito-mode
+---
+description:
+While the names may imply otherwise, 'Incognito mode' and 'private tabs' do not actually protect the identity of the user. They erase all the information on your machine relating to the browsing session after they are closed, but have no measures in place to hide your acivity or digital fingerprint online. This means that an observer can collect your traffic just as easily as any regular browser.
```

Comment:
I'd prefer to be more strong and state that: "While the names may imply otherwise, 'Incognito mode' and 'private tabs' don't make you anonymous on the Internet. "